fixes #5 - Added method to access loklak search API

djmgit opened the pull request:
fixed issue #5, now the library contains method to access
loklak search API service.

sudheesh001 commented on lib/loklak.rb:
```diff
@@ -24,4 +24,31 @@ def self.status()
   end
+
+  def self.search(query=nil, since_date=nil, until_date=nil, from_user=nil, count=nil)
+    method_path = 'search.json?'
```
Remove the `?` from here and put it before `query=` That makes more sense than having a query endpoint with a `?` trailing it.

sudheesh001 commented on lib/loklak.rb:
```diff
@@ -24,4 +24,31 @@ def self.status()
+        method_path += '%20until:' + until_date
+      end
+      if from_user != nil
+        method_path += '%20from:' + from_user
```
Please use ` ` in place of `%20` You can encode the URLs using ruby's libraries

djmgit commented:
@sudheesh001 made the changes

djmgit commented:
@daminisatya @sudheesh001 should I make any other changes in this PR?

mariobehling commented:
Please resolve conflicts.

djmgit committed:
fixes #5 - Added method to access loklak search API

fixed issue #5, now the library contains method to access
loklak search API service

djmgit commented:
@mariobehling resolved conflicts
